pom.xml

```diff
@@ -2586,6 +2591,100 @@
         <zookeeper.version>3.4.6</zookeeper.version>
         <curator.version>2.6.0</curator.version>
       </properties>
+      <dependencyManagement>
+        <dependencies>
+          <!--
+          the AWS module pulls in jackson; its transitive dependencies can create
+          intra-jackson-module version problems.
+          -->
+          <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-aws</artifactId>
```

Comment:
I don't follow, because neither instance uses `<dependencyManagement>`. The child POM already declares a `<dependency>` I don't see the need for duplication here.